### ISSUE TITLE
[Feature Request] Carousel - Displaying partial parts of the next and the previous slide

### ISSUE THREAD

**Issue by ashishb** [vuetifyjs/vuetify#11972]
### Problem to solve
https://vuetifyjs.com/en/components/carousels/#carousels seems to only show one full slide at a time. 

### Proposed solution
It would look more beautiful from a UX perspective if it can also be configured to display a portion of the next and the previous slide.

**Comment by ozgurrgul** [comment]
Indeed, that would be very nice!

**Comment by pagerussell** [comment]
Any update on this feature request? Or any workarounds?

**Comment by J-Sek** [comment]
I don't think it is possible with VCarousel based on Vue transitions - those trigger only for appear/disappear. Internals would need to be overhauled.

Simple carousel [demo](https://play.vuetifyjs.com/playgrounds/YkoXKw) (similar to OwlCarousel or Slick) can be created on top of VSlideGroup. I have decided to replace next/prev with custom easing using VueUse helpers. Auto-scroll would be trivial to add, but looping (aka inifinite scrolling) might be a challenge.

[same demo](https://play.vuetifyjs.com/playgrounds/Nadabw) with `scroll-snap` for reference. No control over easing, but probably has the least overhead.

Another [example](https://play.vuetifyjs.com/playgrounds/kDBFZw) with active slide tracking

**Comment by J-Sek** [comment]
already shipped [in v0](https://0.vuetifyjs.com/components/semantic/carousel#peek), so it most probably arrive in Vuetify v5.0.0

**Comment by J-Sek** [comment]
Instead of waiting for v5:
- use `@vuetify/v0` to create [fully featured](https://0.vuetifyjs.com/components/semantic/carousel#examples) carousel
  - note v0 is shipped as direct dependency since v4.2.0
- or upgrade to v4.3.0 and use revamped VSliderGroup ([example](https://dev.vuetifyjs.com/en/components/slide-groups/#image-slider), does not include drag&drop)
- [VInfiniteCarousel](https://dev.vuetifyjs.com/en/components/infinite-carousels/) is going to ship in v4.3.0 as well - when used with `paused` and `draggable` combined it may also be a good fit (but won't snap items)
